src/thriftpyi/stubs.py

```diff
@@ -154,9 +160,26 @@ def _make_enums(interface: TModuleProxy) -> list[ast.ClassDef]:
     return [item.as_ast(bases=["IntEnum"]) for item in interface.get_enums()]
 
 
-def _make_structs(interface: TModuleProxy, strict: bool) -> list[ast.ClassDef]:
+def _dataclass_decorator(*, frozen: bool, kw_only: bool) -> ast.expr:
+    kws = []
+    if frozen:
+        kws.append(ast.keyword("frozen", ast.Constant(True)))
+    if kw_only:
+        kws.append(ast.keyword("kw_only", ast.Constant(True)))
+    return (
+        ast.Name("dataclass", ast.Load())
+        if not kws
+        else ast.Call(ast.Name("dataclass", ast.Load()), [], kws)
```

Comment:
This is to give the generated code a more "hand written" look, but we could simplify this and just have it do `@dataclass()` in the empty case. Let me know.